plot-lib/plot/private/no-gui/plot2d.rkt

```diff
@@ -68,7 +70,16 @@
                     [plot-legend-anchor  legend-anchor])
        (define area (make-object 2d-plot-area%
                       bounds-rect x-ticks x-far-ticks y-ticks y-far-ticks legend dc x y width height))
-       (plot-area area renderer-list))]))
+       (plot-area area renderer-list)
+
+       (define bounds (vector (vector (assert (ivl-min (vector-ref bounds-rect 0)) real?)
```

Comment:
I think the bounds should be calculated and memory allocated for them in the `get-plot-bounds` method below, perhaps lazily initialize them as a member of the returned plot-metrics object.  

The `plot/dc` method is used in every 2D plot and most of these uses will just discard the plot-metrics object  so I want to avoid the memory allocation penalty (and runtime execution penalty) in all those use cases.